open

```diff
@@ -27,18 +27,15 @@ done
 
 cd "${projectRoot}"
 
-# Frontends
-code ./fcp-fd-landing-page;
-code ./fcp-fd-messages;
-
-# Backends
```

Comment:
Some scripts had these comments e.g. `#Frontends`, `#Backends`, `#Customer` etc. Don't really think we need this if I'm honest and we weren't really applying this across all scripts. Instead I've arranged all scripts to be in alphabetical order.